lib/elixir_console_web/live/console_live.ex

```diff
@@ -38,6 +34,8 @@ defmodule ElixirConsoleWeb.ConsoleLive do
     Sandbox.terminate(sandbox)
   end
 
+  def print_prompt, do: "> "
```

Comment:
Not feeling strong about this, but I consider this function should not be located in this module.

We can create a `lib/elixir_console_web/live/console_live/helpers.ex` or similar to hold this function (I know it would feel like a short module, but the intention of being a shared thing would be clearer).

Reply:
I agree with you, it fits better in its own helpers module